### PR TITLE
TW1433 Sending requests/variables refresh

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/sending-requests/variables.md [hunk 1]
```diff
@@ -100,6 +101,18 @@ In order from broadest to narrowest, these scopes are: _global_, _collection_, _
 <!-- -->
 > Postman stores variables as strings. If you store objects or arrays, remember to `JSON.stringify()` them before storing, and `JSON.parse()` them when you retrieve them.
 
+## Initial and current values
+
+When you edit variables, each one has an _Initial value_ and _Current value_:
+
+* **Initial value** is a value that's set in the element (collection, environment, or globals) where the variable is defined. This value is synced to Postman's servers, and is shared with your team when you share that element.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/variables.md:108]: We can also mention that users can use initial value for collaborating with teammates.

File: src/pages/docs/sending-requests/variables.md [hunk 2]
```diff
@@ -100,6 +101,18 @@ In order from broadest to narrowest, these scopes are: _global_, _collection_, _
...
+
+* **Current value** is used when sending a request. These are local values, and aren't synced to Postman's servers. If left empty, the current value automatically uses the initial value. If you change a current value, it won't be persisted in the original shared collection, environment, or globals.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/variables.md:110]: Current value doesn't assume initial value when left empty

File: src/pages/docs/sending-requests/variables.md [hunk 3]
```diff
@@ -100,6 +101,18 @@ In order from broadest to narrowest, these scopes are: _global_, _collection_, _
...
+
+Initial value can be useful when sharing elements with others, but it's important to remember that sensitive data in an initial value will also be shared with others, and potentially with the world. Be careful setting initial values, and consider using secret variables to mask sensitive data.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/variables.md:112]: Let's keep this close to the initial value section itself

**Review by giridharvc7 — CHANGES_REQUESTED** [review]

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! Do you need to change the `updated` date on any of these files?

**Pull request merged**